bci_tester/data.py

```diff
@@ -717,15 +720,38 @@ def create_BCI(
     )
 ]
 
-TOMCAT_9_CONTAINER, TOMCAT_10_CONTAINER = [
+APACHE_TOMCAT_10_CONTAINERS = [
     create_BCI(
-        build_tag=f"{APP_CONTAINER_PREFIX}/tomcat:{tomcat_ver}",
+        build_tag=f"{SAC_CONTAINER_PREFIX}/apache-tomcat:10-jre{jre_version}",
+        bci_type=ImageType.SAC_APPLICATION,
+        available_versions=("15.6",),
+        forwarded_ports=[PortForwarding(container_port=8080)],
+    )
+    for jre_version in (21,)
+] + [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/apache-tomcat:10-jre{jre_version}",
         bci_type=ImageType.APPLICATION,
```

Comment:
```suggestion
        bci_type=ImageType.SAC_APPLICATION,
```

Reply:
that's the tumbleweed container, I intentionally didn't set it to SAC_APPLICATION.